db/migrate/20211207132134_rename_contributions_url.rb

```diff
@@ -0,0 +1,5 @@
+class RenameContributionsUrl < ActiveRecord::Migration[5.2]
+  def change
+    rename_column :projects, :contributions_url, :contributing_url
```

Comment:
Can you redo the migrations so they are just one migration, if you run `rake db:rollback` twice it should undo both migrations, then you can remove this file and fix the column name in the other migration file then run `rake db:migrate` again to update the schema.